integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -250,8 +250,9 @@ public void exec(int depth, int concurrencyDepth, TachyonURI path) throws Except
 
   /**
    * The authenticate user is gotten from current thread local. If MasterInfo starts a concurrent
-   * thread to do operations, AuthorizedClientUser will be null. So AuthorizedClientUser.set()
-   * should be called in the Callable.call to set this user for testing.
+   * thread to do operations, AuthorizedClientUser will be null. So
```

Comment:
@jsimsa Thanks for the comments. AuthorizeClientUser is done. The link to MasterInfo would not work, because it is part of Mesos (as far as I know) which is not linked in the integration tests.